File: win-prog/ProgWinVB/Hello Windows Forms/FormProperties/FormProperties.vb
'-----------------------------------------------
' FormProperties.vb (c) 2002 by Charles Petzold
'-----------------------------------------------
Imports System.Drawing
Imports System.Windows.Forms

Module FormProperties
    Sub Main()
        Dim frm As New Form()

        frm.Text = "Form Properties"
        frm.BackColor = Color.BlanchedAlmond
        frm.Width *= 2
        frm.Height \= 2
        frm.FormBorderStyle = FormBorderStyle.FixedSingle
        frm.MaximizeBox = False
        frm.Cursor = Cursors.Hand
        frm.StartPosition = FormStartPosition.CenterScreen

        Application.Run(frm)
    End Sub
End Module

File: win-prog/ProgWinCS/Hello Windows Forms/FormProperties/FormProperties.cs
//---------------------------------------------
// FormProperties.cs © 2001 by Charles Petzold
//---------------------------------------------
using System.Drawing;
using System.Windows.Forms;

class FormProperties
{
     public static void Main()
     {
          Form form = new Form();

          form.Text              = "Form Properties";
          form.BackColor         = Color.BlanchedAlmond;
          form.Width            *= 2;
          form.Height           /= 2;
          form.FormBorderStyle   = FormBorderStyle.FixedSingle;
          form.MaximizeBox       = false;
          form.Cursor            = Cursors.Hand;
          form.StartPosition     = FormStartPosition.CenterScreen;

          Application.Run(form);
     }
}
